### PR TITLE
Add Astrian/tg-watchdog

### PR DESCRIPTION
**Pull request by Astrian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ This is a curated list of projects that are using grammY. Anyone is welcome to t
 - [bot-base/scan-tool-bot](https://github.com/bot-base/scan-tool-bot) - Scan QR codes with your camera. [ᴜsᴇ](https://t.me/ScanToolBot)
 - [ShinkarenkoMaxim/simple-quiz](https://github.com/ShinkarenkoMaxim/simple-quiz) - A simple quiz bot as Web App. [ᴜsᴇ](https://t.me/yaqb_bot)
 - [dcdunkan/show-json-bot](https://github.com/dcdunkan/show-json-bot) - See the JSON data behind Telegram messages. [ᴜsᴇ](https://t.me/jsoonbot)
+- [Astrian/tg-watchdog](https://github.com/Astrian/tg-watchdog) - The brand-new solution for anti-abuse from Telegram groupchats. [ᴜsᴇ](https://t.me/WatchdogVerifyBot?startgroup=start&admin=can_invite_users)
```
Comment (roj1512) [inline README.md:35]:
```suggestion
- [Astrian/tg-watchdog](https://github.com/Astrian/tg-watchdog) - Anti-abuse solution for Telegram groups. [ᴜsᴇ](https://t.me/WatchdogVerifyBot?startgroup=start&admin=can_invite_users)
```

Comment (roj1512) [inline README.md:35]: `can_invite_users`?

Comment (Astrian) [reply README.md:35]: It suggests the Telegram app request the user to add the bot to the group and select `can_invite_user` permission as default.

Comment (roj1512) [reply README.md:35]: Why would it need that permission?

Comment (Astrian) [reply README.md:35]: This permission is related to Join Requests, which allow the hot to receive alerts of new Join Requests, chat with candidates actively, and approve or decline their requests. And that's the only permission the bot need.

Detail: https://core.telegram.org/bots/api-changelog#december-7-2021

**Review by roj1512 — APPROVED** [review]

**Pull request merged**